setup.py

```diff
@@ -26,8 +26,9 @@
         'Programming Language :: Python :: 3.7',
     ],
     license='BSD',
+    package_data={'keg_storage': ['py.typed']},
     packages=find_packages(),
-    zip_safe=True,
+    zip_safe=False,
```

Comment:
This is disabled here so that mypy will be able to find the package (in the [mypy docs](https://mypy.readthedocs.io/en/latest/installed_packages.html), this is mentioned in one of the notes).

Or do we need to continue supporting `zip_safe=True`?